packages/countries/scripts/generateCountrySymbol.mjs

```diff
@@ -149,71 +158,92 @@ const generateCountrySymbolComponents = ({
           },
         },
       ],
-    });
+    };
+
+    const optimizedSvg = optimize(svgString, optimizeOptions);
+    const optimizedSharpSvg = optimize(sharpSvgString, optimizeOptions);
 
     const getUidString = (countryCode, value) => {
       return `\$\{uid\}-${countryCode}-${value}`;
     };
 
-    const svgPaths = optimize(optimizedSvg.data, {
```

Comment:
This diff looks more chunky than it actually is.

To avoid excessive duplication, the `mapHTMLAttributesToReactProps` and `removeSvg` have been moved out of the `optimize()` call and into a separate object. This is because they are duplicated in `const svgPaths = optimize(...` and `const sharpSvgPaths = optimize(...` but the `find-viewBox` options are different for each instance *(and the order of the plugin declaration is important: `find-viewBox` must go between `mapHTMLAttributesToReactProps` and `removeSvg`)*